codegen/smithy-aws-go-codegen/src/main/java/software/amazon/smithy/aws/go/codegen/AwsGoDependency.java

```diff
@@ -81,6 +81,6 @@ protected static GoDependency module(
     }
 
     private static final class Versions {
-        private static final String AWS_SDK = "v0.0.0-20201006075021-8b185f9d6dff";
+        private static final String AWS_SDK = "v0.0.0-20201014161145-592e24615bd4";
```

Comment:
Should this be using the pseudo versions, `v0.26.1-0.20201014161145-592e24615bd4` ?

Reply:
Seems like the pseudo version was only required for `Smithy-go` dependency.